salesforce/bulk.go

```diff
@@ -430,3 +399,100 @@ func (c *Connector) getIncompleteJobResults(jobInfo *GetJobInfoResult) *JobResul
 
 	return jobResult
 }
+
+func (c *Connector) BulkDelete(ctx context.Context, params BulkOperationParams) (*BulkOperationResult, error) {
+	jobBody := map[string]any{
+		"operation": "delete",
+		"object":    params.ObjectName,
+	}
+
+	return c.BulkOperation(ctx, params, jobBody)
+}
+
+//nolint:funlen,cyclop
+func (c *Connector) BulkOperation(
```

Comment:
This should not be exported, this should just be an internal method (not capitalized)